docs/concepts/multiaddr.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Multi-addresses'
+description: Multi-addresses and how they are used in IPFS
+---
+
+# Understanding Multi-Addresses in IPFS
+
+A [multiaddr](https://github.com/multiformats/multiaddr) encodes multiple layers of addressing information into a single "future-proof" path structure.
+
+It is a self-describing format that allows multiple protocols encoded in a single string. This allows for flexibility in addressing and routing network traffic in a decentralized system like IPFS.
```

Comment:
Suggestion: could consolidate these two points and remove the phrase "future-proof".  Also, I think there is a grammar error here:

```
A multi-address (multiaddr) is a self-describing format that allows multiple protocols to be encoded into a single string. This allows for flexibility in addressing and routing network traffic in a decentralized system like IPFS.
```